docs/index.rst

```diff
@@ -21,4 +21,5 @@ Starknet SDK for Python
    guide
    api
    development
+   devnet_utils
```

Comment:
nit: I think it would be slightly better if we move it above `api` and below `guide`